### ISSUE TITLE
Build failure on nightly Rust

### ISSUE THREAD

**Issue by TsarFox** [racer-rust/racer#1074]
Hi,

`cargo +nightly install racer` fails with the following:

```
error[E0046]: not all trait items implemented, missing: `mixed_site`
   --> /home/jakob/.cargo/registry/src/github.com-1ecc6299db9ec823/rustc-ap-syntax-583.0.0/ext/proc_macro_server.rs:657:1
    |
657 | impl server::Span for Rustc<'_> {
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ missing `mixed_site` in implementation
    |
    = note: `mixed_site` from trait: `fn(&mut Self) -> <Self as proc_macro::bridge::server::Types>::Span`
```

This is on `rustc 1.40.0-nightly (2e7244807 2019-10-04)`, and the version of `rustc-ap-syntax` being brought in is `v583.0.0`.

**Comment by TsarFox** [comment]
My apologies, not this crate's fault.

**Comment by xttjsn** [comment]
Did you manage to resolve the problem?

**Comment by TsarFox** [comment]
Yes :]  As a temporary fix, you can apply #1076 and `cargo install`
